setup/index.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
```

Comment:
Don’t we need this line, since `index.js` uses `let`? I couldn’t get it to run otherwise.